src/components/asthma/views/AsthmaHeartAndLungsView/AsthmaHeartAndLungsView.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useState } from 'react';
+import { Layout, NavigationBar, Title } from '../../../presentational';
+import { DailyDataProvider, registerDailyDataProvider, simpleAvailabilityCheck } from '../../../../helpers/query-daily-data';
+import { daytimeBloodOxygenLevelDataProvider, daytimeRestingHeartRateDataProvider, nighttimeBloodOxygenLevelDataProvider, nighttimeRestingHeartRateDataProvider, respiratoryRateDataProvider } from '../../helpers/daily-data-providers';
+import { AsthmaAlertTakeoverNotice, AsthmaBarChart } from '../../components';
+import { useInitializeView } from '../../../../helpers/Initialization';
+import { asthmaDataService, isBloodOxygenLevelWithinRange, isDaytimeRestingHeartRateWithinRange, isNighttimeRestingHeartRateWithinRange, isRespiratoryRateWithinRange } from '../../helpers';
+import language from '../../../../helpers/language';
+import { randomDataProvider } from '../../../../helpers/daily-data-providers';
+
+const DaytimeRestingHeartRateDailyDataType = 'Asthma.DaytimeRestingHeartRate';
+const NighttimeRestingHeartRateDailyDataType = 'Asthma.NighttimeRestingHeartRate';
+const RespiratoryRateDailyDataType = 'Asthma.RespiratoryRate';
+const DaytimeBloodOxygenLevelDailyDataType = 'Asthma.DaytimeBloodOxygenLevel';
+const NighttimeBloodOxygenLevelDailyDataType = 'Asthma.NighttimeBloodOxygenLevel';
+
+registerDailyDataProvider(DaytimeRestingHeartRateDailyDataType, daytimeRestingHeartRateDataProvider, simpleAvailabilityCheck('Project', ['DaytimeRestingHeartRate']));
```

Comment:
There's no cost to "registering" a daily data provider and this feels awkward to have to declare in order for a view to work?  I would probably chuck this code into asthma-data or something that's imported at a higher level?

Reply:
Done.  Moved to `ashtma-data`.